plugins/state_history_plugin/tests/session_test.cpp

```diff
@@ -530,7 +530,7 @@ BOOST_FIXTURE_TEST_CASE(test_session_fork, state_history_test_fixture) {
       std::vector<eosio::state_history::block_position> have_positions;
       eosio::state_history::state_result result;
       // we should get 4 consecutive block result
-      for (int i = 0; i < 4; ++i) {
+      for (uint32_t i = 0; i < 4; ++i) {
```

Comment:
`size_t` is likely better here.

Reply:
Hum, I'm not sure. The warning was when comparing with `r.this_block->block_num` which is a `uint32_t`.
If it was a comparison with `<container>.size()` then I would have used `size_t`.

Reply:
Ah, I thought it was likely complaining about the `written_data[i]`. `uint32_t` then.